src/picongpu/include/simulation_defines/param/laserConfig.param

```diff
@@ -74,6 +74,13 @@ namespace picongpu
         /* laser phase shift (no shift: 0.0) */
         BOOST_CONSTEXPR_OR_CONST float_X LASER_PHASE = 0.0; /* unit: rad, periodic in 2*pi */
 
+        /* Use only the 0th Laguerremode for a standard Gaussian */
+        BOOST_CONSTEXPR_OR_CONST uint32_t MODENUMBER = 0;
+        PMACC_CONST_VECTOR(float_X, MODENUMBER + 1, LAGUERREMODES, 1.0);
+        /* This is just an example for a more complicated set of Laguerre modes. */
```

Comment:
Maybe you also want to have a look at [sympy](http://www.sympy.org/en/index.html) in case you have to do some symbolic solves on the way.